src/go/rpk/pkg/cli/root.go

```diff
@@ -92,6 +92,7 @@ func Execute() {
 		return opts, cobra.ShellCompDirectiveNoSpace
 	})
 
+	root.Version = version.Pretty()
```

Comment:
Nit: can you move this to the struct initialization on line 71?

Reply:
done